event_listener/dfusion_db/snapp_events/tests/test_withdraw_request.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from unittest.mock import Mock
+
+from event_listener.dfusion_db.models import Withdraw
+from ..withdraw_request import WithdrawRequestReceiver
+
+EMPTY_STATE_HASH = "0x00000000000000000000000000000000000000000000000000000000000000"
```

Comment:
Can we import this from some common test_util file instead of copying it 6 times?

Reply:
Yes sir. Just a moment.